packages/sonarwhal/src/lib/utils/misc.ts

```diff
@@ -344,6 +355,10 @@ const getSonarwhalPackage = () => {
     return require(path.join(__dirname, '../../../../package.json'));
 };
 
+const sameOrigin = (url1: string, url2: string): boolean => {
```

Comment:
Wasn't this done in another PR?

Reply:
Was changed on your request (see: https://github.com/sonarwhal/sonarwhal/pull/916#discussion_r176550487)

Reply:
OK, I missed the notification.
Then those changes shouldn't be there, right?

Reply:
I think git mess up after a rebase, this code shouldn't be there.
For some reason, I have an extra commit in my PR (`Chore: Remove `same-origin` dependency`) and it shouldn't be there